align_anything/datasets/text_image_video_to_text/__init__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2024 PKU-Alignment Team. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Dataset classes for text to text training."""
```

Comment:
text to text

Reply:
Fixed in https://github.com/PKU-Alignment/align-anything/pull/76/commits/c8f2df0e58f74cb73712af6f33bee07dc6a9ac20